resources/views/components/stars.blade.php

```diff
@@ -0,0 +1,32 @@
+@props(['score' => 0, 'count' => null])
+
+<div {{ $attributes->twMerge('flex items-center gap-1') }}>
+    <div class="relative">
+        <div class="flex gap-0.5">
+            @for ($star = 0; $star < 5; $star++)
+                <x-heroicon-s-star class="size-5 text-muted/50" />
+            @endfor
+        </div>
+        <div
+            style="width: {{ $score }}%"
+            @if ($attributes->has('v-bind:score'))
+                v-bind:style="{ width: ({{ $attributes['v-bind:score'] }} || 0) + '%'}"
+            @endif
+            class="absolute inset-0 flex gap-0.5 overflow-hidden"
+        >
```

Comment:
Could this be written cleaner with `@attributes()`? See: https://github.com/rapidez/blade-directives#attributes or is this the cleanest option?